plugins/backfill/storage/transformerExporter.go

```diff
@@ -3,60 +3,7 @@
-	clip_yfi_a_v1_5_0 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/clip/initializers/yfi_a/v1_5_0"
-	clip_zrx_a_v1_6_0 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/clip/initializers/zrx_a/v1_6_0"
-	dog_v1_3_0 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/dog/initializers/v1_3_0"
-	flap_v1_0_9 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/flap/initializers/v1_0_9"
-	flop_v1_0_9 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/flop/initializers/v1_0_9"
-	jug "github.com/makerdao/vdb-mcd-transformers/transformers/storage/jug/initializer"
-	median_aave_v1_2_2 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_aave/v1_2_2"
-	median_bal_v1_1_14 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_bal/v1_1_14"
-	median_bat_v1_0_0 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_bat/v1_0_0"
-	median_comp_v1_1_2 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_comp/v1_1_2"
-	median_eth_v1_0_0 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_eth/v1_0_0"
-	median_knc_v1_0_8 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_knc/v1_0_8"
-	median_link_v1_1_2 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_link/v1_1_2"
-	median_lrc_v1_1_2 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_lrc/v1_1_2"
-	median_mana_v1_0_9 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_mana/v1_0_9"
-	median_matic_v1_9_4 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_matic/v1_9_4"
-	median_uni_v1_2_1 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_uni/v1_2_1"
-	median_usdt_v1_0_4 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_usdt/v1_0_4"
-	median_wbtc_v1_0_6 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_wbtc/v1_0_6"
-	median_wsteth_v1_9_8 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_wsteth/v1_9_8"
-	median_yfi_v1_1_14 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_yfi/v1_1_14"
-	median_zrx_v1_0_8 "github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers/median_zrx/v1_0_8"
-	pot "github.com/makerdao/vdb-mcd-transformers/transformers/storage/pot/initializer"
-	spot "github.com/makerdao/vdb-mcd-transformers/transformers/storage/spot/initializer"
-	vow "github.com/makerdao/vdb-mcd-transformers/transformers/storage/vow/initializer"
+	vat "github.com/makerdao/vdb-mcd-transformers/transformers/storage/vat/initializer"
```

Comment:
Can we create this as a new and separate transformer rather than modifying the existing one? Essentially, copy-paste the existing `vat` transformer to `vat_lite`, remove the storage keys, and treat it as a separate module. It could exist in parallel to the standard `vat` transformer and would only be used in this specific backfill situation.